README.md

```diff
@@ -6,7 +6,8 @@ To learn more about Cerberus, please visit the [Cerberus website](http://enginee
 
 This library is compatible with node 8.x +, for previous versions of Node use the [1.x versions of the client](https://github.com/Nike-Inc/cerberus-node-client/tree/1.X_kms_client)
```

Comment:
We aren't building for node 8 it seems, so is this comment out of date?

Reply:
fixed